src/Avalonia.Base/Avalonia.Base.csproj

```diff
@@ -31,6 +31,7 @@
     <InternalsVisibleTo Include="Avalonia.Skia.RenderTests, PublicKey=$(AvaloniaPublicKey)" />
     <InternalsVisibleTo Include="Avalonia.Skia.UnitTests, PublicKey=$(AvaloniaPublicKey)" />
     <InternalsVisibleTo Include="Avalonia.UnitTests, PublicKey=$(AvaloniaPublicKey)" />
+    <InternalsVisibleTo Include="Avalonia.Win32, PublicKey=$(AvaloniaPublicKey)" />
```

Comment:
Is this needed?

Reply:
Because i using cache [here](https://github.com/AvaloniaUI/Avalonia/pull/8460/files#diff-85a4c6a7618dca0962a3495fd093c7297500ce8c81e7ccbd67c92b7c78e3bd56).